homeWork2/getArrayPokemons.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+async function getArrayPokemons() {
+  const response = await fetch(`https://pokeapi.co/api/v2/pokemon`);
+  const objectResults = await response.json();
+  const arrayInfoPokemons = objectResults.results;
+  const arrayPokemons = arrayInfoPokemons.map((pokemon) => pokemon.name);
+  console.log(arrayPokemons);
+}
```

Comment:
when you are working with async functions, you have to wrap your code into `try catch`